apps/organisations/models.py

```diff
@@ -94,6 +97,45 @@ class Organisation(models.Model):
     def __str__(self):
         return self.name
 
+    @cached_property
+    def projects(self):
+        return Project.objects \
+            .filter(organisation=self,
+                    is_archived=False,
+                    is_draft=False)
+
+    def get_projects_list(self, user):
+        active_projects = []
+        future_projects = []
+        past_projects = []
+
+        projects = query.filter_viewable(self.projects, user)
```

Comment:
What is the difference to self.projects? Is it sth. from the rules, so that also private projects are displayed?